packages/ag-charts-website/src/content/docs/error-series/index.mdoc

```diff
@@ -1,5 +1,37 @@
 ---
 title: 'Error Series'
+enterprise: true
 ---
 
-TODO
+Error bars show the uncertainty of continuous data values. They can be enabled
+on Line, Bar and Scatter Series.
+
+## Line Series Error Bars
+
+{% chartExampleRunner title="Line Series Error Bars" name="line-series-error-bars" type="generated" options="{ \"enterprise\": true }" /%}
+
+Error bars can be enabled on Line Series using the `errorBars` key.
+
+The `yLowerKey` and `yUpperKey` defines the keys used to retrieve the error
+values from the data.
```

Comment:
I'm inclined to use the word "dataset" instead of "data" to clarify that it's actual statistical data rather than config data like theme options. Thoughts?